pkgs/development/libraries/nlohmann_json/default.nix

```diff
@@ -24,7 +24,8 @@ in stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [ cmake ];
 
   cmakeFlags = [
-    "-DBuildTests=${if finalAttrs.doCheck then "ON" else "OFF"}"
+    "-DJSON_BuildTests=${if finalAttrs.doCheck then "ON" else "OFF"}"
+    "-DJSON_FastTests=ON"
```

Comment:
presumable this skips some slow tests? Are they valuable for us? I assume if we skip them for all platforms they are not super important.

Reply:
For ``JSON_FASTTESTS``: https://github.com/nlohmann/json/blob/dae54122684bbe256117b82992e95e7840720b75/tests/CMakeLists.txt#L4
Just skips some expensive and slow tests. I have no idea if they're valuable to us or not, but I doubt it.
For ``JSON_BUILDTESTS``: IIRC this is just the actually correct flag to build tests.

Reply:
It'd be interesting to know what the speed increases are here

Reply:
I do know it's required for risc-v on visionfive at least, I can just make it a ``lib.optionals`` if that's preferable.

Reply:
> It'd be interesting to know what the speed increases are here

Fast tests take ~2 seconds, slow tests are ~120-190 sec plus building them. I would vote to always only run the fast tests.